.github/workflows/build-nodejs.yml

```
@@ -0,0 +1,250 @@
+            name=$(basename "$dir")
+            platform="${name#bindings-}"
+            for file in "$dir"/*.node; do
+              if [ -f "$file" ]; then
+                cp "$file" "release/lni_js.${platform}.node"
+              fi
+            done
+          done
+          ls -la release/
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)